2007-12-18  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/posix/getaddrinfo.c (default_labels): order
	the labels by value.

Index: sysdeps/posix/getaddrinfo.c
===================================================================
RCS file: /cvs/glibc/libc/sysdeps/posix/getaddrinfo.c,v
retrieving revision 1.121
diff -u -d -p -r1.121 getaddrinfo.c
--- sysdeps/posix/getaddrinfo.c	20 Nov 2007 00:40:41 -0000	1.121
+++ sysdeps/posix/getaddrinfo.c	18 Dec 2007 17:56:06 -0000
@@ -1110,6 +1110,10 @@ static const struct prefixentry default_
 			  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01 } } },
       128, 0 },
     { { .in6_u
+	= { .u6_addr8 = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+			  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 } } },
+      0, 1 },
+    { { .in6_u
 	= { .u6_addr8 = { 0x20, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 			  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 } } },
       16, 2 },
@@ -1140,11 +1144,7 @@ static const struct prefixentry default_
     { { .in6_u
 	= { .u6_addr8 = { 0x20, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 			  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 } } },
-      32, 7 },
-    { { .in6_u
-	= { .u6_addr8 = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-			  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 } } },
-      0, 1 }
+      32, 7 }
   };
 
 
